src/tests/ftest/soak/soak.py

```diff
@@ -222,22 +226,37 @@ def get_remote_logs(self):
         # copy the files from the remote
         # TO-DO: change scp
         this_host = socket.gethostname()
-        rsync_str = "rsync -avtr --min-size=1B"
+        cmd = [
+            "bash", "-c",
+            "\"rsync -avtr --min-size=1B {0} {1}:{0}/..".format(
+                self.test_log_dir, this_host)
+        ]
         result = slurm_utils.srun(
-            NodeSet.fromlist(self.hostlist_clients),
-            "bash -c \"{0} {1} {2}:{1}/.. && rm -rf {1}/*\"".format(
-                rsync_str, self.test_log_dir, this_host),
-            self.srun_params)
+            NodeSet.fromlist(self.hostlist_clients), cmd, self.srun_params)
         if result.exit_status == 0:
-            cmd = "cp -R -p {0}/ \'{1}\'; rm -rf {0}/*".format(
-                self.test_log_dir, self.outputsoakdir)
-            try:
-                result = process.run(cmd, shell=True, timeout=30)
-            except process.CmdError as error:
+            rm_cmd = ["rm", "-rf", "{}/*".format(self.test_log_dir)]
```

Comment:
Do we need a full path for the `rm` command?

Reply:
not needed